composer.json

```diff
@@ -0,0 +1,5 @@
+{
+    "require": {
+        "globalpayments/php-sdk": "3.0.*"
```

Comment:
The SDK repo and Packagist do not have this tag, so this should cause issues on install. We should make sure a matching tag exists on the repo/Packagist.

Are Drupal Commerce 8 modules distributed via Composer? If so, we'll need to expand this file out to include the necessary bits required to list a package. If needed, our [Omnipay plugin](https://github.com/hps/omnipay-heartland/blob/master/composer.json) can be used for reference.

Reply:
Modules can be installed with or without composer.  The Commerce Kickstart distro requires composer and that's what I tested this with.  I've added more information using the Omnipay file for reference.

Reply:
Looks fantastic.